common/app/views/mainAMP.scala.html

```diff
@@ -4,10 +4,10 @@
 <!doctype html>
 <html AMP>
     <head>
+        @fragments.metaData(metaData, true)
```

Comment:
good but we still don't quite get utf-8 to the top... https://github.com/guardian/frontend/blob/master/common/app/views/fragments/metaData.scala.html#L16